Cask

```diff
@@ -7,4 +7,4 @@
 (depends-on "markdown-mode")
 
 (package-file "lsp-ui.el")
-(files "*.el")
+(files "*.el" "lsp-ui-doc.html")
```

Comment:
I am not sure whether this is sufficient but I suspect that melpa does not take this into account but uses only this file https://github.com/melpa/melpa/blob/master/recipes/lsp-ui